available-hardware.mdx

```diff
@@ -48,7 +48,7 @@ Once again, you only pay for what you need!
 ## Storage
 
 We provide you with a persistent storage volume attached to your deployment.
-You can use this storage volume to store any data that you need to persist between deployments. Accessing your persistent storage is covered in depth for [cortex here](./cerebrium/data-sharing-storage/persistent-storage).
+You can use this storage volume to store any data that you need to persist between deployments. Accessing your persistent storage is covered in depth for [cortex here](/cerebrium/data-sharing-storage/persistent-storage).
```

Comment:
that path doesn't look correct lmao

Reply:
Changed it to an absolute path because the relative paths are susceptible to breaks if this file gets moved.